CITATION.cff

```diff
@@ -3,8 +3,8 @@ message: If you use this software, please cite it using the metadata from this f
 type: software
 title: 'brick: Building sector model with heterogeneous renovation and construction
   of the stock'
-version: 0.5.9
-date-released: '2024-10-09'
+version: 0.5.10
```

Comment:
I was wondering if this could be a "minor change" instead of a "bugfix/patch", meaning that this would be version 0.6.0. Of course the code hasn't changed much, but I would expect the results to differ?
Still I think both would be ok, so if you prefer this to be only a very minor version update, I'm fine with that, too.

Reply:
Actually, results haven't changed much and I consider this a first step of a potentially bigger overhaul. So I'd still go with 3rd-digit increment.